packages/docs/docs/advanced/canvas-operations.mdx

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 5
+slug: /canvas-operations
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Canvas Operations
+
+Because Motion Canvas is build ontop of the Browser's 2D Rendering Context, we
+can make use several canvas operations that are provided by the Browser.
+
+## Filters
+
+<AnimationPlayer name="canvas-operations-filters" />
+
+Every Component accepts a list of filters. You can find a list of all filters
+the Browser can understand on
+[MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/filter).
+
+All filters available in Motion Canvas can be found under
+`@motion-canvas/2d/lib/partials`
+([API Documentation](/api/2d/partials#Functions)). Note that some filters, like
+`opacity`, can be set directly on the [`Node`](/api/2d/components/Node#opacity),
+and as a result are not exported from [partials](/api/2d/partials#Functions).
+
+`filters` accepts an array. This means that you can stack multiple filters onto
+a single Node. The order in which you add them does matter, as can be seen
+below:
+
+<AnimationPlayer name="canvas-operations-filters-stacking" />
```

Comment:
I know that the animation includes code snippets but it's impossible to copy them.
Having an example in a code block together with imports would be nice.

Reply:
The relevant code bit is just above it. So I would argue that you could just copy that instead. 
The comment above `compositeOperation` tells you what changing between source-in and source-out does.

I could add the relevant imports.
![image](https://user-images.githubusercontent.com/27686785/228870338-e22abbd0-251b-466f-9b4e-10a9d3c4c702.png)

Reply:
I must have missed that code block.
Btw, we now have a fiddle component that displays an editor with a preview which could be better in this case:

https://user-images.githubusercontent.com/64662184/236676320-6753910d-d506-4c0f-8593-56d3b8f233a2.mp4

You could make the reader edit the operations themselves and toggle between a few presets.
It's also really easy to write in markdown:
https://github.com/motion-canvas/motion-canvas/blob/fae72b3729f2a7b5b1291a92fbf2ff8e767e0d43/packages/docs/docs/components/spline.mdx?plain=1#L362-L418